templates/managed-namespaces-gateways.yaml

```diff
@@ -31,7 +31,13 @@ istio:
         app: istio-ingressgateway
         istio: {{ $namespace.name }}-ingressgateway
       autoscaleEnabled: true
-      autoscaleMin: 2
+      # minimum of 1 per AZ. According to
+      # https://kubernetes.io/docs/setup/best-practices/multiple-zones/#pods-are-spread-across-zones
+      # pods are spread across "failure zones" by default. NLB cross-zone routing isn't in this version
+      # of kubernetes and as we've enabled `externalTrafficPolicy: Local` connections to the NLB in an AZ
+      # that has no ingressgateway results in timeouts. Once k8s 1.14 lands the following annotation solves it
```

Comment:
I mean, we could just upgrade to 1.14, it's available...

Reply:
I've noted this hack on the EKS upgrade card

Reply:
The correcting annotation has also already been applied. We thought it best to test and unbreak sandbox before embarking on the lengthy process of the 1.14 upgrade.